cmake/compiler_flags_Intel_Fortran.cmake

```diff
@@ -1,40 +1,44 @@
-# (C) Copyright 2020-2020 NOAA/NWS/NCEP/EMC
+# (C) Copyright 2009-2016 ECMWF.
 #
 # This software is licensed under the terms of the Apache Licence Version 2.0
 # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation nor
+# does it submit to any jurisdiction.
 
 ####################################################################
 # FLAGS COMMON TO ALL BUILD TYPES
 ####################################################################
 
-set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -g -traceback")
+set( CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -D_REAL8_")
```

Comment:
@srherbener do you know why the *normal* Fortran flags would have `-g -traceback` for iodaconv these should be in `DEBUG` no?

Reply:
I agree with applying the `-traceback` option for only the debug build. The tracebacks from the optimized builds are usually unintelligible.